SecurityManager/Core/Domain/AccessControl/IAccessControlSettings.cs

```diff
@@ -1,26 +1,24 @@
-// This file is part of re-strict (www.re-motion.org)
+﻿// This file is part of re-strict (www.re-motion.org)
 // Copyright (c) rubicon IT GmbH, www.rubicon.eu
-// 
+//
 // This program is free software; you can redistribute it and/or modify
-// it under the terms of the GNU Affero General Public License version 3.0 
+// it under the terms of the GNU Affero General Public License version 3.0
 // as published by the Free Software Foundation.
-// 
-// This program is distributed in the hope that it will be useful, 
-// but WITHOUT ANY WARRANTY; without even the implied warranty of 
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 // GNU Affero General Public License for more details.
-// 
+//
 // You should have received a copy of the GNU Affero General Public License
 // along with this program; if not, see http://www.gnu.org/licenses.
-// 
+//
 // Additional permissions are listed in the file re-motion_exceptions.txt.
-// 
-using System;
-using Remotion.SecurityManager.Domain.OrganizationalStructure;
-
-namespace Remotion.SecurityManager.UnitTests.Configuration
+//
+namespace Remotion.SecurityManager.Domain.AccessControl
 {
-  public class TestOrganizationalStructureFactory : OrganizationalStructureFactory
+  public interface IAccessControlSettings
```

Comment:
xml doc. please also add the threadsafety-xmldoc attribute. I also like to add a `<seealso cref=` pointing to the implementation. See examples in source code / it's added after the other doc-tags:)